tests/components/electric_kiwi/conftest.py

```diff
@@ -27,6 +27,17 @@
 type ComponentSetup = Callable[[], Awaitable[bool]]
 
 
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
```
There are not a lot of tests where you don't need this